<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manager_mainmenu</class>
 <widget class="QWidget" name="manager_mainmenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图书管理员菜单</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>10</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>身份:  manager</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>80</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>280</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出登录</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>341</width>
     <height>451</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="title">
    <string>图书管理</string>
   </property>
   <widget class="QPushButton" name="delete_book">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除图书</string>
    </property>
   </widget>
   <widget class="QPushButton" name="insert_book">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>添加图书</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update_book">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>修改图书信息</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>341</width>
     <height>421</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="title">
    <string>查询模块</string>
   </property>
   <widget class="QPushButton" name="query_book">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>查询图书信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="query_borrow">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>查询借阅信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="query_manager">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>查询个人信息</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>341</width>
     <height>541</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="title">
    <string>服务模块</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="borrow_book">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>图书借阅</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selete_opinion">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>待处理反馈</string>
    </property>
   </widget>
   <widget class="QPushButton" name="return_book">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>图书归还</string>
    </property>
   </widget>
   <widget class="QPushButton" name="extend_time">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>借阅延期</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="manager_name">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>80</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="change_password">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>160</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
